src/build/command_replacements.go

```diff
@@ -192,7 +192,7 @@ func replaceSequenceLabel(state *core.BuildState, target *core.BuildTarget, labe
 }
 
 func checkAndReplaceSequence(state *core.BuildState, target, dep *core.BuildTarget, in string, runnable, multiple, dir, outPrefix, hash, test, allOutputs, tool bool) string {
-	if allOutputs && !multiple && len(dep.Outputs()) != 1 {
+	if allOutputs && !multiple && len(dep.Outputs()) > 1 {
```

Comment:
I think it should still panic for when there are no outputs; might be nice to special-case that though so the message is more explicit.

Reply:
There is another case checking `runnable` and `len(dep.Outputs()) == 0` as a special case later actually! :)